reactivesocket-tck-drivers/src/main/java/io/reactivesocket/tckdrivers/client/JavaClientDriver.java

```diff
@@ -0,0 +1,517 @@
+/*
+ * Copyright 2016 Facebook, Inc.
```

Comment:
The Facebook one should be on all these files. Copyrights are whoever submits/edits/touches a file, so there can be multiple at the top of the file.

If someone doesn't care about their copyright, then they can "assign" it to the default Netflix, but we use the Facebook copyright. 

We are however adopting the Apache 2 license since that's what this project uses.